docker/bootstrap/Dockerfile.common

```diff
@@ -1,3 +1,11 @@
+FROM node:16-bullseye-slim as node
```

Comment:
This is using `bullseye`, but then we're using `buster` for the Go image. I don't think we should mix different bistro versions to avoid risk of mismatches in binary symbols etc.